src/styles/blocks/header.scss

```diff
@@ -0,0 +1,74 @@
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-direction: row;
+
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
```

Comment:
Is it the separate BEM block? Looks like it should not be in the file for header styles